src/ipnet.rs

```diff
@@ -262,8 +259,9 @@ impl Ipv4Net {
         Ipv4Addr::from(addr | !self.mask())
     }
 
-    /// Return the first host address within this subnet; for subnets wider
-    /// than a /31 or /32 that is the first address after the network address.
+    /// Return the first host address within this subnet. For /31 and /32
+    /// subnets that is the first address after; for wider subnets this returns
```

Comment:
This phrase seems to be missing something: "that is the first address after <??>"

Reply:
fixed